Description: <cp testing.sh to /usr/bin>
 
 cp testing.sh to /usr/bin

 sl (5.02-3) focal; urgency=medium
Author: brandenwu <raranhot@hotmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2021-04-30

--- sl-5.02.orig/testing.sh
+++ sl-5.02/testing.sh
@@ -1,2 +1,3 @@
 #!/bin/bash
 echo "this is a test from Branden Wu"
+
